_topic_map.yml

```diff
@@ -2069,65 +2069,36 @@ Name: Service Mesh
 Dir: service_mesh
 Distros: openshift-enterprise,openshift-webscale
 Topics:
-- Name: Service Mesh release notes
-  File: servicemesh-release-notes
-- Name: Service Mesh architecture
-  Dir: service_mesh_arch
-  Topics:
-  - Name: Understanding Service Mesh
-    File: understanding-ossm
-  - Name: Understanding Kiali
-    File: ossm-kiali
-  - Name: Understanding Jaeger
-    File: ossm-jaeger
-  - Name: Comparing Service Mesh and Istio
+- Name: Service Mesh 2.x
```

Comment:
Before we merge this into the production pre-2.0, I think we need to remove the 2.0 folder.

Reply:
Commented out.